server/db/db.connect.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+const toJson = require('@meanie/mongoose-to-json');
+mongoose.plugin(toJson);
+

```

Comment:
what is this toJson plugin for?

Reply:
toJson plugin is used to get the normalized JSON output.
Mongodb by default it outputs `id` as `_id` and if we want normalized output like `id` you can use this plugin. Now you don't have to change the code to `id` to `_id` as your default JSON output will be `id` only. 
Hope it makes sense.

Here is the reference - [link](https://www.npmjs.com/package/@meanie/mongoose-to-json)